bindings.c

```diff
@@ -3951,6 +3951,9 @@ static void __attribute__((constructor)) collect_subsystems(void)
 			goto out;
 		*p2 = '\0';
 
+		if (!strcmp(p, ""))
```

Comment:
Can you add a brief comment here (and identical at the next) explaning why, for future debuggers?